res/cmake/def.cmake

```diff
@@ -11,6 +11,10 @@ endif() # clang
 set (PHARE_LINK_FLAGS )
 set (PHARE_BASE_LIBS )
 
+if(NOT DEFINED PHARE_MPIRUN_POSTFIX)
```

Comment:
without this, running multiple instances of mpirun -n 2 python3 ... effectively only lets run one process, the rest are descheduled until the first is finished

we could not default it here but set it manually in TC builds, or have it as an option defaulted to ON

Reply:
this was moved to the configurator, so to rm